tests/unit/data-viewer.spec.ts

```diff
@@ -0,0 +1,191 @@
+          },
+        });
+
+        const firstHeaderCellWrapper = wrapper.find('.data-viewer__header-cell');
+        firstHeaderCellWrapper.trigger('click');
+        expect(firstHeaderCellWrapper.classes()).toContain('data-viewer__header-cell--active');
+      });
+
+      it('should remove selection on an active column', () => {
+        const dataset = [
+          { columnA: 'value1', columnB: 'value2', columnC: 'value3' },
+          { columnA: 'value4', columnB: 'value5', columnC: 'value6' },
+          { columnA: 'value7', columnB: 'value8', columnC: 'value9' },
+          { columnA: 'value10', columnB: 'value11', columnC: 'value12' },
+          { columnA: 'value13', columnB: 'value14', columnC: 'value15' },
+        ];
+        const wrapper = shallowMount(DataViewer, {
+          propsData: {
+            dataset: dataset,
+          },
+        });
+
+        const firstHeaderCellWrapper = wrapper.find('.data-viewer__header-cell');
+        // Select Column
+        firstHeaderCellWrapper.trigger('click');
+        expect(firstHeaderCellWrapper.classes()).toContain('data-viewer__header-cell--active');
+        // Unselect Column
+        firstHeaderCellWrapper.trigger('click');
+        expect(firstHeaderCellWrapper.classes()).not.toContain('data-viewer__header-cell--active');
+      });
+    });
+  });
+
+  describe('body', () => {
+    it('should have 5 rows', () => {
+      const dataset = [
+        { columnA: 'value1', columnB: 'value2', columnC: 'value3' },
+        { columnA: 'value4', columnB: 'value5', columnC: 'value6' },
+        { columnA: 'value7', columnB: 'value8', columnC: 'value9' },
+        { columnA: 'value10', columnB: 'value11', columnC: 'value12' },
+        { columnA: 'value13', columnB: 'value14', columnC: 'value15' },
+      ];
+      const wrapper = shallowMount(DataViewer, {
+        propsData: {
+          dataset: dataset,
+        },
+      });
+
+      const rowsWrapper = wrapper.findAll('.data-viewer__row');
+      expect(rowsWrapper.length).toEqual(5);
+    });
+
+    it('should pass down the right value to DataViewerCell', () => {
+      const dataset = [
+        { columnA: 'value1', columnB: 'value2', columnC: 'value3' },
+        { columnA: 'value4', columnB: 'value5', columnC: 'value6' },
+        { columnA: 'value7', columnB: 'value8', columnC: 'value9' },
+        { columnA: 'value10', columnB: 'value11', columnC: 'value12' },
+        { columnA: 'value13', columnB: 'value14', columnC: 'value15' },
+      ];
+      const wrapper = shallowMount(DataViewer, {
+        propsData: {
+          dataset: dataset,
+        },
+      });
+
+      const firstRowWrapper = wrapper.find('.data-viewer__row');
+      const firstCellWrapper = firstRowWrapper.find('dataviewercell-stub');
+      // Syntax specific to stubbed functional Vue Component
+      expect(firstCellWrapper.attributes('value')).toEqual('value1');
+    });
+  });
+
+  describe('when clicking on first head cell', () => {
```

Comment:
This is more of a `context` (starts with when)
Otherwise it's describing the row selection behavior (`describe('row selection)`)

Reply:
Jest doesn't provide `context` keyword like mocha :/

Reply:
So ok to rename